source/docs/software/commandbased/organizing-command-based.rst

```diff
@@ -249,7 +262,37 @@ If we want to avoid the verbosity of adding required subsystems as parameters to
 
   .. code-block:: c++
 
-    // TODO
+    class AutoRoutines {
+      public:
+        AutoRoutines(Drivetrain *drivetrain, Intake *intake)
+          : drivetrain{drivetrain}, intake{intake} {}
+
+        frc2::CommandPtr DriveAndIntake() {
+            return frc2::cmd::Sequence(
+                frc2::cmd::Parallel(
+                    drivetrain->DriveCommand(0.5, 0.5),
+                    intake->RunIntakeCommand(1.0)
+                ).WithTimeout(5.0),
+                frc2::cmd::Parallel(
+                  drivetrain->StopCommand();
+                  intake->StopCommand();
+                )
+            );
+        }
+
+        frc2::CommandPtr DriveThenIntake() {
+            return frc2::cmd::Sequence(
+                drivetrain->DriveCommand(0.5, 0.5).WithTimeout(5.0),
+                drivetrain->StopCommand(),
+                intake->RunIntakeCommand(1.0).WithTimeout(5.0),
+                intake->StopCommand()
+            );
+        }
+
+      private:
+        Drivetrain *drivetrain;
+        Intake *intake;
+    }
```

Comment:
Applies to everywhere: C++ has a semicolon after the closing brace of class definitions.